src/components/organisms/about/about.scss

```
@@ -0,0 +1,29 @@
+@use "../../variables" as *;
+
+.about {
+  margin-bottom: 100px;
+  @media screen and(max-width: $breakpoint-lg) {
+    margin-bottom: 40px;
+  }
+  & .about-img-wrapper {
```

```suggestion
   .about-img-wrapper {
```
子孫セレクタは＆がなくても問題なく解釈されます。（他の部分も同様）

---

一応消しました。